### PR TITLE
Package tensorflow.0.0.11

### PR DESCRIPTION
**Pull request by LaurentMazare**
### `tensorflow.0.0.11`
TensorFlow bindings for OCaml
The tensorflow-ocaml project provides some OCaml bindings for TensorFlow, a
machine learning framework. These bindings are in an early stage of their
development. Some operators are not supported and the API is likely to change
in the future. You may also encounter some segfaults. That being said they
already contain the necessary to train a convolution network using various
optimizers.

---
* Homepage: https://github.com/LaurentMazare/tensorflow-ocaml
* Source repo: git+https://github.com/LaurentMazare/tensorflow-ocaml.git
* Bug tracker: https://github.com/LaurentMazare/tensorflow-ocaml/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/tensorflow/tensorflow.0.0.11/opam [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  "stdio"
+]
+
+depopts: [
+  "camlimages"
+  "gnuplot"
+  "npy"
+]
+
+available: [ os != "macos" ]
```
[the remark this answers is not in the document]
Comment (LaurentMazare) [reply packages/tensorflow/tensorflow.0.0.11/opam]: Right, windows antd bsd won't work. For macos I don't have a way to test it for now so I've enabled it and we'll see if it passes CI.

[the remark this answers is not in the document]
Comment (LaurentMazare) [reply packages/tensorflow/tensorflow.0.0.11/opam]: The macos build failed CI so I deactivated it for the time being.

[the remark this answers is not in the document]
Comment (mseri) [reply packages/tensorflow/tensorflow.0.0.11/opam]: I see. Yes, i seems to need the gnu linker. I will also try on my laptop with a newer version of osx.

**Comment by mseri** [comment]
Thanks

**Pull request merged**